.github/workflows/release.yml

```diff
@@ -0,0 +1,37 @@
+name: Release
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js 12.x
+        uses: actions/setup-node@v1
+        with:
+          node-version: 12.x
+      - run: npm ci
+      - run: npm run build
+      - name: Set up Python
+        uses: actions/setup-python@v1
+        with:
+          python-version: "3.5"
```

Comment:
Should we use a newer python version, End Of Life for Python 3.5: 2020-09-13

Reply:
I chose Python 3.5 since that's our minimum version currently in imjoy-elfinder. It's good practice to use the minimum version when testing code.

I think we could already now update minimum version to Python 3.6. What do you think?